src/Components/ImagesTable/ImageLinkDirect.js

```diff
@@ -10,145 +9,115 @@ import {
   TextVariants,
 } from '@patternfly/react-core';
 import { DownloadIcon, ExternalLinkAltIcon } from '@patternfly/react-icons';
-import { resolveRelPath } from '../../Utilities/path';
 
-const ImageLinkDirect = (props) => {
-  const navigate = useNavigate();
+const ImageLinkDirect = ({ imageType, uploadStatus, uploadOptions }) => {
   const fileExtensions = {
     vsphere: '.vmdk',
     'guest-image': '.qcow2',
     'image-installer': '.iso',
   };
 
-  const uploadStatus = props.imageStatus
-    ? props.imageStatus.upload_status
-    : undefined;
-  if (uploadStatus) {
```

Comment:
GitHub makes it very difficult to see, but I've just removed the if from here and aligned the code.
Only other change is removal of the `props.`